bin/start_webserver

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+
+# Path to the application root.
+APP_ROOT = File.expand_path('..', __dir__)
+
+# We want to guard against failures at each stage of this
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+FileUtils.chdir APP_ROOT do
+  # TODO: We're going to want to remove this evenutally!!
+  # Recreates the database on each deployment.
+  system! 'DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bundle exec bin/rails db:migrate:reset db:seed'
+
+  # Firstly we're going to run the migrations.
+  # We do this via this helper script to sleep and retry on conurrent failures
+  system! 'bundle exec bin/rails runner lib/run_migrations_with_concurrent_guard.rb'
```

Comment:
So every time we deploy we run the migrations as the service starts. 

- If the migrations fail, then this script will abort, the server (below) won't start, and so the deploy will fail (based on the Fargate health checks). We will also raise to Bugsnag here eventually.
- If two servers run migrations at the same time (very likely) the first will lock it and the second will keep getting an exception until the first has finished. Once that's done, the second will run the migrations again (a no-op) then continue. There's a script below for this.